### PR TITLE
add two joe vaults

### PR DESCRIPTION
**Pull request by cartman-dev**
EGG-AVAX JLP -- Updated
Vault: 0x491E09Eb736c0c4CD85DF694A22D1C0dec8c4C25
Strategy: 0x580dc7caCe89b63B2A2B6b05CC8a36711CF853A6
Want: 0x3052a75dfD7A9D9B0F81E510E01d3Fe80A9e7ec7
PoolId: 32

USDC-AVAX LP
Vault: 0x7E5bC7088aB3Da3e7fa1Aa7ceF1dC73F5B00681c
Strategy: 0x351FFF63542538DED81Ec61C7cC01D51EB08f884
Want: 0xf4003F4efBE8691B60249E6afbD307aBE7758adb
PoolId: 50

### CODE REVIEW

**Comment by roman-monk** [comment]
FEED liquidity is on Pangolin, so Egg strat has to use 2 routers

**Comment by cartman-dev** [comment]
> FEED liquidity is on Pangolin, so Egg strat has to use 2 routers

Understood, I will re-deploy a suitable strategy.  See [beefy-contracts PR #167](https://github.com/beefyfinance/beefy-contracts/pull/167)

File: src/features/configure/vault/avalanche_pools.js [hunk 1]
```diff
@@ -34,6 +34,77 @@ export const avalanchePools = [
...
+      'PLATFORM_ESTABLISHED',
+      'CONTRACTS_VERIFIED',
+    ],
+    stratType: 'StratLP',
+    withdrawalFee: '0%',
+    addLiquidityUrl:
+      'https://traderjoexyz.com/pool/0x3052a75dfd7a9d9b0f81e510e01d3fe80a9e7ec7/AVAX',
+    buyTokenUrl:
+      'https://traderjoexyz.com/trade?outputCurrency=0x7761e2338b35bceb6bda6ce477ef012bde7ae611',
+    createdAt: 1646420486,
```
Comment (cartman-dev) [inline src/features/configure/vault/avalanche_pools.js:106]:
```suggestion
    createdAt: 1646787205,
```

File: src/features/configure/vault/avalanche_pools.js [hunk 2]
```diff
@@ -34,6 +34,77 @@ export const avalanchePools = [
...
+      'COMPLEXITY_LOW',
+      'BATTLE_TESTED',
+      'IL_LOW',
+      'MCAP_LARGE',
+      'PLATFORM_ESTABLISHED',
+      'CONTRACTS_VERIFIED',
+    ],
+    stratType: 'StratLP',
+    addLiquidityUrl:
+      'https://traderjoexyz.com/pool/0xf4003f4efbe8691b60249e6afbd307abe7758adb/AVAX',
```
Comment (cartman-dev) [inline src/features/configure/vault/avalanche_pools.js:67]:
```suggestion
      'https://traderjoexyz.com/farm/0xf4003F4efBE8691B60249E6afbD307aBE7758adb-0x188bED1968b795d5c9022F6a0bb5931Ac4c18F00',
```

File: src/features/configure/vault/avalanche_pools.js [hunk 3]
```diff
@@ -34,6 +34,77 @@ export const avalanchePools = [
...
+      'BATTLE_TESTED',
+      'IL_HIGH',
+      'MCAP_MICRO',
+      'PLATFORM_ESTABLISHED',
+      'CONTRACTS_VERIFIED',
+    ],
+    stratType: 'StratLP',
+    withdrawalFee: '0%',
+    addLiquidityUrl:
+      'https://traderjoexyz.com/pool/0x3052a75dfd7a9d9b0f81e510e01d3fe80a9e7ec7/AVAX',
```
Comment (cartman-dev) [inline src/features/configure/vault/avalanche_pools.js:103]:
```suggestion
      'https://traderjoexyz.com/farm/0x3052a75dfD7A9D9B0F81E510E01d3Fe80A9e7ec7-0x188bED1968b795d5c9022F6a0bb5931Ac4c18F00',
```

**Pull request merged**